src/common/utils/string-utils.ts

```diff
@@ -0,0 +1,3 @@
+export function convertUnicodeToAscii(str: string): string {
```

Comment:
this is to remove accents from accentuated characters? maybe rename it to reflect that
```
removeAccentsFromString
```

Reply:
I think the name is correct here. This is not only for accents, it's just converting unicode to ascii so it's always readable and prevent spoofing.